.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+  release:
+    types: [ published, created, edited ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - uses: jakejarvis/s3-sync-action@master
+        name: Deploy non-tag branches
+        if: github.event_name == 'pull_request'
+        with:
+          args: --acl public-read --follow-symlinks --delete --cache-control "max-age=60"
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
```

Comment:
The bucket isn't secret, it is `pixi.js`

Reply:
@englercj I know that buckets are unique per AWS account, therefore it's not trivial to test it on your fork.

Reply:
```
AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET || 'pixi.js' }}
```
should be better

Reply:
Ah you're using the secrets as a way to parameterize the bucket. That's probably fine.